cpp/merkletree/sparse_merkle_tree.h

```diff
@@ -171,29 +172,25 @@ class SparseMerkleTree {
   // or the leaf index is 0.
   //
   // @param path the path of the leaf whose inclusion proof to return.
-  std::vector<std::string> InclusionProof(const Path& path);
+  std::vector<std::string> InclusionProof(const Path& path) const;
 
   std::string Dump() const;
 
  private:
-  // WARNING WARNING WARNING
-  // 64 < 256 !
-  // WARNING WARNING WARNING
-  // TODO(alcutter): BIGNUM probably.
-  typedef uint64_t IndexType;
+  typedef cert_trans::BigNum IndexType;
```

Comment:
bignums are an obvious way to do this, but quite a lot of baggage to store what is really an array of bits that you never do any real arithmetic on?